Library/Formula/nedit.rb

```diff
@@ -0,0 +1,39 @@
+class Nedit < Formula
+  homepage "http://www.nedit.org"
+  url "https://downloads.sourceforge.net/project/nedit/nedit-source/nedit-5.6-src.tar.gz"
+  sha1 "133641224b0969e31e3404b874c71c35341f028b"
+
+  depends_on :x11
+  depends_on "openmotif"
```

Comment:
Interesting I can't find it either. It seems that I used the formula from here:

https://gist.github.com/steakknife/60a39a32ae84e12238a2

to install openmotif, and this formula is not yet in.

What do I do with this then? Options are:

1. Get the openmotif formula into the main Homebrew repo first, and then do nedit one next
2. Redo the nedit formula to use lesstif (which is in the repo)